packages/cfd/src/Containers/mt5-mobile-redirect-option.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Button, Icon, StaticUrl, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { getDownloadOptions } from '../Helpers/constants';
+import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
+import './mt5-mobile-redirect-option.scss';
+
+type TMT5MobileRedirectOptionProps = {
+    mt5_trade_account: DetailsOfEachMT5Loginid;
+};
+const MT5MobileRedirectOption = ({ mt5_trade_account }: TMT5MobileRedirectOptionProps) => {
+    const mobileDownloadOptions = getDownloadOptions({ mt5_trade_account }).filter(
+        option => option.device === 'mobile'
+    );
+
+    return (
+        <div className='mt5-download-container'>
+            {mobileDownloadOptions.map(option => (
+                <Button
+                    key={option.icon}
+                    className={classNames('mt5-download-container--option', {
+                        blue: option.highlight,
+                    })}
+                    classNameSpan='mt5-download-container--span'
+                    onClick={() => {
+                        if (option.highlight) {
+                            window.location.replace(option.href as string);
+                        }
+                        window.open(option.href as string, '_blank');
+                    }}
+                >
+                    <div className='center'>
```

Comment:
do we really need to have this class. If we do let's give it something more meaningful than `center`.